acme_diags/libs/cdms2.py

```diff
@@ -0,0 +1,8 @@
+import os
```

Comment:
This file will replace all `import cdms2` with `from acme_diags.libs.cdms2 import cdms2`